### ISSUE TITLE
colorbox zoomooz conflict 

### ISSUE THREAD

**Issue by naeluh** [jackmoore/colorbox#360]
Hi, 

I have this fiddle - http://jsfiddle.net/naeluh/bCtpe/ - which includes 2 plugins 

I have this fiddle that I listed here that has the zoomooz which basically is supported by this js file http://theconfluencegroup.com/new_site5/js/jquery.zoomooz.js which us included in the external sources part of the fiddle. If I remove this file zoomooz stops working obviously because it does not have the js to support its functions but now after that file is removed colorbox works again the if you click the divs below the photos/videos those are now active colorbox modal windows that can be triggered. 

Basically what I am asking here is there a way to get these 2 plugins to work together ?
Is there some way to alter colorbox to work and avoid the conflict ?

Please let me know if I can provide any other information to help solve this issue 

thanks
